dthpool.sol

```diff
@@ -0,0 +1,284 @@
+/*
+This file is part of the DAO.
+
+The DAO is free software: you can redistribute it and/or modify
+it under the terms of the GNU lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The DAO is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU lesser General Public License for more details.
+
+You should have received a copy of the GNU lesser General Public License
+along with the DAO.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+contract DAOProxy {
+    function proposals(uint _proposalID) returns(
```

Comment:
Just leaving this here as a note.

This DAO Proxy contract has the `proposals() as a function like this because importing the normal DAO contract and calling `proposals()` on the DAO to access the struct members will not work due to [this](https://github.com/ethereum/solidity/issues/598) solidity bug.

Until this bug is fixed, such a hack is necessary.